pkg/web_app/lib/src/widget/downloads_chart/widget.dart

```diff
@@ -18,7 +20,33 @@ void create(HTMLElement element, Map<String, String> options) {
   final data = WeeklyVersionDownloadCounts.fromJson((utf8.decoder
       .fuse(json.decoder)
       .convert(base64Decode(dataPoints)) as Map<String, dynamic>));
-  drawChart(svg, data);
+
+  final weeksToDisplay = math.min(28, data.totalWeeklyDownloads.length, 28);
+
+  final majorDisplayLists = prepareWeekLists(
+    data.totalWeeklyDownloads,
+    data.majorRangeWeeklyDownloads,
+    weeksToDisplay,
+  );
+  final majorRanges = data.majorRangeWeeklyDownloads.map((e) => e.versionRange);
+
+  // final minorDisplayLists = prepareWeekLists(
+  //   data.totalWeeklyDownloads,
+  //   data.minorRangeWeeklyDownloads,
+  //   weeksToDisplay,
+  // );
+  // final minorRanges = data.minorRangeWeeklyDownloads.map((e) => e.versionRange);
+
+  // final patchDisplayLists = prepareWeekLists(
+  //   data.totalWeeklyDownloads,
+  //   data.patchRangeWeeklyDownloads,
+  //   weeksToDisplay,
+  // );
+  // final patchRanges = data.patchRangeWeeklyDownloads.map((e) => e.versionRange);
```

Comment:
Is this needed?

Reply:
I meant - please don't commit commented out code. Either delete it until it is needed, or un-comment it.

Reply:
I'll remove it then.